src/Altinn.Profile.Integrations/Migration/v0.04/03-setup-table.sql

```diff
@@ -0,0 +1,39 @@
+-- Scrips generated by "dotnet ef migrations script --idempotent --startup-project ../Altinn.Profile"
+
+-- Create table Organizations
+CREATE TABLE organization_notification_address.organizations (
+    registry_organization_id integer GENERATED BY DEFAULT AS IDENTITY,
+    registry_organization_number character varying(9) NOT NULL,
+    CONSTRAINT organization_id_pkey PRIMARY KEY (registry_organization_id)
+);
+
+-- Create table RegistrySyncMetadata
+CREATE TABLE organization_notification_address.registry_sync_metadata (
+    last_changed_id character varying(32) NOT NULL,
+    last_changed_date_time timestamp with time zone NOT NULL,
+    CONSTRAINT registry_sync_metadata_pkey PRIMARY KEY (last_changed_id)
+);
+
+-- Create table NotificationsAddresses
+CREATE TABLE organization_notification_address.notifications_address (
+    contact_info_id integer GENERATED BY DEFAULT AS IDENTITY,
+    registry_organization_id integer NOT NULL,
+    registry_id character varying(32) NOT NULL,
+    address_type integer NOT NULL,
+    domain character varying(200) NOT NULL,
+    address character varying(200) NOT NULL,
+    full_address character varying(200) NOT NULL,
+    created_date_time timestamp with time zone NOT NULL DEFAULT (now() at time zone 'utc'),
```

Comment:
I never stopped to ask why this has a default value.

Reply:
Responding to myself. Looks like it's not from "kofuvi". Altinn 2 is setting it in code.